packages/vector_graphics_compiler/bin/vector_graphics_compiler.dart

```diff
@@ -1,15 +1,35 @@
 import 'dart:io';
 import 'dart:typed_data';
 
+import 'package:args/args.dart';
 import 'package:vector_graphics_compiler/vector_graphics_compiler.dart';
 
+final ArgParser argParser = ArgParser()
+  ..addOption('input',
+      abbr: 'i',
+      help: 'The path to a file containing a single SVG',
+      mandatory: true)
+  ..addOption(
+    'ouput',
+    abbr: 'o',
+    help:
+        'The path to a file where the resulting vector_graphic will be written.\n'
+        'If not provided, defaults to <input-file>.vg',
+  );
+
 Future<void> main(List<String> args) async {
-  if (args.length != 2) {
-    print('Usage: vector_graphics_compiler input.svg output.bin');
+  final ArgResults results;
+  try {
+    results = argParser.parse(args);
+  } on FormatException catch (err) {
+    print(err.message);
+    print(argParser.usage);
     exit(1);
   }
-  final String xml = File(args[0]).readAsStringSync();
-  final File outputFile = File(args[1]);
+  final String inputFilePath = results['input'] as String;
+  final String xml = File(inputFilePath).readAsStringSync();
+  final File outputFile =
+      File(results['output'] as String? ?? '$inputFilePath.vg');
```

Comment:
```suggestion
      File(results['output'] as String? ?? '${inputFilePath.replaceAll('.svg', '.vg'}');
```

Reply:
what if the input file is named `foo.svgProcessing.svg`, then we'd write `foo.vgProcessing.vg` its not wrong exactly but appending is safer if the user doesn't care what the output is exactly.

Reply:
Yeah, fair. You could try to make it just replace the last .svg I guess but it's not worth the complexity.